app/main.py

```diff
@@ -1,6 +1,23 @@
+import requests
+from pprint import pprint
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+
+    url = "http://api.weatherapi.com/v1/current.json"
```

Comment:
Code Style issue: As per the checklist, URLs or any other string that does not change over time should be declared as a constant. Constants should be declared at the top of the file and their names should be in uppercase.